frontend/src/app/shared/components/file-links/storage-file-list-item/storage-file-list-item.html

```diff
@@ -0,0 +1,32 @@
+<label
+  class="spot-list--item-action"
+>
+  <spot-checkbox
+    [checked]="selected"
+    (click)="changeSelection.emit()"
```

Comment:
It's better if you listen for `checkedChange` here, because a click on the wrapping label will also change the checked state here, and I'm unsure if that triggers a click event.